src/main/java/teammates/common/datatransfer/questions/FeedbackTextResponseDetails.java

```diff
@@ -3,7 +3,6 @@
 import teammates.common.util.SanitizationHelper;
 
 public class FeedbackTextResponseDetails extends FeedbackResponseDetails {
-
```

Comment:
This change seems unrelated?
Let's try to avoid changes in files that don't impact your PR.